sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeFileClient.cs

```diff
@@ -2210,6 +2222,21 @@ public new DataLakeFileClient WithCustomerProvidedKey(Models.DataLakeCustomerPro
         /// <param name="leaseId">
         /// Lease id for operation.
         /// </param>
+        /// <param name="leaseAction">
+        /// Lease action.
+        /// <see cref="LeaseAction.Acquire"/> will attempt to aquire a new lease on the file, with <see cref="DataLakeFileAppendOptions.ProposedLeaseId"/> as the lease ID.
```

Comment:
NIT typo - find and replace "aquire" with "acquire".

Reply:
fixed.